### PR TITLE
Pond wrapper

### PR DESCRIPTION
**Pull request by ankita0805**
Merging all the updates so far for Pond to flatten branch as requested by Joey.

### CODE REVIEW

**Comment by joyliu37** [comment]
You cal remove all the csv file inside.

File: lake/utils/util.py
Comment (mbstrange2) [inline lake/utils/util.py:513]: Can all of this be moved to the Pond file? This code should be attached to an instance of a Pond object

File: lake/utils/wrapper_lake.py
Comment (mbstrange2) [inline lake/utils/wrapper_lake.py:13]: the wrapper function should just take a premade object and wrap it, not decide which constructor to call

File: tests/test_pond.py
Comment (mbstrange2) [inline tests/test_pond.py:236]: can we opt for passing the name in or figuring it out without if/else everywhere for the dsl flag?

**Review by mbstrange2 — CHANGES_REQUESTED** [review]
Let's try to make things clean and prevent redundant code, otherwise looks good.

File: lake/utils/util.py
Comment (kavyasreedhar) [reply lake/utils/util.py:513]: @ankita0805

File: lake/utils/wrapper_lake.py
Comment (kavyasreedhar) [reply lake/utils/wrapper_lake.py:13]: sure, I can refactor that.

File: tests/test_pond.py
Comment (kavyasreedhar) [reply tests/test_pond.py:236]: yeah, I'll do it at the top if dsl is true and just use one name throughout so it's cleaner.

Comment (kavyasreedhar) [reply tests/test_pond.py]: done.

File: lake/utils/wrapper_lake.py
Comment (kavyasreedhar) [reply lake/utils/wrapper_lake.py]: Done.

**Review by kavyasreedhar — APPROVED** [review]

**Review by mbstrange2 — APPROVED** [review]
lgtm once it passes

**Comment by kavyasreedhar** [comment]
Need to make sure garnet and AHA still work before merging.

**Pull request merged**
